### ISSUE TITLE
lista de revisão

### ISSUE THREAD

**Issue by DarkShadow96-cpu** [professorlineu/PRCE2-2019-2#2]
testa ai

https://github.com/professorlineu/PRCE2-2019-2/tree/master/Atividades/Felipe_Alexandre/revisão

**Comment by Ri-lang** [comment]
Senhor Felipe Alexandre, corrigir a atividade 02 do capítulo 11, incrementando o GetLine.

**Comment by JoaoPauloGomez** [comment]
Os códigos estão funcionais, só achei que poderia adicionar linhas de comentario para deixar mais explicito o que foi feito no quinto código, sétimo e oitavo também!

**Comment by DarkShadow96-cpu** [comment]
Sim, é um mal abito meu de não coloca muitos comentários  vô tenta corrigir em atividades futuras valeu

**Comment by DarkShadow96-cpu** [comment]
Sim, é um mal abito meu de não coloca muitos comentários  vô tenta corrigir em atividades futuras valeu
